scripts/dev/sminer-init

```diff
@@ -7,4 +7,4 @@ export TRUST_PARAMS=1
 tag=${TAG:-debug}
 
 go run -tags=$tag ./cmd/lotus wallet import ~/.genesis-sectors/pre-seal-t01000.key
-go run -tags=$tag ./cmd/lotus-miner init --actor=t01000 --genesis-miner --pre-sealed-sectors=~/.genesis-sectors --pre-sealed-metadata=~/.genesis-sectors/pre-seal-t01000.json
+go run -tags=$tag ./cmd/lotus-storage-miner init --actor=t01000 --genesis-miner --pre-sealed-sectors=~/.genesis-sectors --pre-sealed-metadata=~/.genesis-sectors/pre-seal-t01000.json
```

Comment:
whai is this? is it "lotu-storage-miner" right?

Reply:
The binary is `lotus-miner` bur directory is `lotus-storage-miner`:
 https://github.com/filecoin-project/lotus/tree/next/cmd